pkg/loader/openapi.go

```diff
@@ -115,6 +116,13 @@ func getOpenAPITools(t *openapi3.T, defaultHost string) ([]types.Tool, error) {
 				toolDesc = toolDesc[:1024]
 			}
 
+			toolName := operation.OperationID
+			if toolName == "" {
+				// When there is no operation ID, we use the method + path as the tool name and remove all characters
+				// except letters, numbers, underscores, and hyphens.
+				toolName = regexp.MustCompile(`[^a-zA-Z0-9_-]+`).ReplaceAllString(strings.ToLower(method)+strings.ReplaceAll(pathString, "/", "_"), "")
```

Comment:
nit: in general, it's bad practice to have things that could panic in code. It is unlikely that this will panic, but I wanted to call this out.

Typically, creating a package variable that is initialized on package initialization (i.e. on startup) is preferable.